src/Servers/Kestrel/shared/test/TestApp.cs

```diff
@@ -49,7 +49,6 @@ public static async Task EchoAppPipeWriter(HttpContext httpContext)
             if (buffer.Length > 0)
             {
                 await request.Body.ReadUntilEndAsync(buffer).DefaultTimeout();
-                await response.StartAsync();
```

Comment:
I don't think we should have unrelated tests hit the inefficient code path. Tests like ChunksWithGetMemoryBeforeFirstFlushStillFlushes should be sufficient.